src/app/ubs-admin/components/ubs-admin-orders/ubs-admin-orders.component.html

```diff
@@ -0,0 +1,119 @@
+<div class="wrapper">
+  <div class="main_header">Ваші замовлення</div>
+  <div *ngIf="isAnyOrders" class="if_empty"><span>Ви ще не зробили замовлень у нас.</span><a href="#">Зробити замовлення</a></div>
+  <div *ngIf="!isAnyOrders" class="cards">
+    <div class="header_list">
+      <div>№ замовлення</div>
+      <div>Статус</div>
+      <div>Сума оплати</div>
+    </div>
+    <div *ngFor="let order of orders" class="card">
+      <div class="empty_card">
+        <img *ngIf="order.extend" src="assets/img/ubs-admin-orders/arrow_down.svg" alt="arrow down" (click)="changeCard(order.id)" />
+        <img *ngIf="!order.extend" src="assets/img/ubs-admin-orders/arrow_up.svg" alt="arrow up" (click)="changeCard(order.id)" />
+        <div>{{ order.order }}</div>
+        <div>{{ order.status }}</div>
+        <div class="order_sum">{{ order.sum }}</div>
+        <div class="btns_group">
+          <div *ngIf="order.status === 'Сформовано'" class="btn_grid_cancel">
```

Comment:
don't use UA lang for coding

Reply:
I added a new property in data object - "order_status". This property can have the values: "formed", "done"...